pkg/apis/nais.io/v1alpha1/application_types.go

```diff
@@ -65,6 +67,20 @@ type ApplicationSpec struct {
 	Logformat string `json:"logformat,omitempty"`
 }
 
+// ApplicationStatus contains different NAIS status properties
+type ApplicationStatus struct {
+	CorrelationID           string `json:"correlationID,omitempty"`
+	DeploymentRolloutStatus string `json:"deploymentRolloutStatus,omitempty"`
+	LastError               Entry  `json:"lastError,omitempty"`
```

Comment:
LastError, LastSuccess and LastSyncedHash is not implemented. Let's start out by removing them from the spec, and then implement them later after discussing their function and contents?

Reply:
Sounds good!